src/_data/resources.json

```diff
@@ -806,6 +837,12 @@
 			"additional": "Michigan, United States",
 			"url": "https://twitter.com/annarbora11y"
 		},
+		{
+			"title": "axe-con",
+			"description": "free, virtual digital accessibility conference with tracks for design, develoopers, business leaders, and compliance",
```

Comment:
Can we capitalize the first word and add a period after the word "compliance"? We try to use full sentences for our descriptions.